back/app/controllers/application_controller.rb

```diff
@@ -46,7 +46,7 @@ def user_not_authorized(exception)
       render json: { errors: { base: [{ error: 'blocked', details: { block_end_at: current_user.block_end_at } }] } },
         status: :unauthorized
     else
-      reason = exception.reason || 'Unauthorized!'
+      reason = exception.try(:reason) || 'Unauthorized!'
```

Comment:
This is a nice pattern. I must _try_ to remember it.

Reply:
Actually, I *try* to use it as little as possible, as it can easily hide bugs.

Reply:
Good point 🤔